ui/src/shared/parsing/showSeries.ts

```diff
@@ -0,0 +1,26 @@
+interface ParseShowSeriesResponse {
+  errors: string[]
+  series: string[]
+}
+
+const parseShowSeries = (response): ParseShowSeriesResponse => {
+  const results = response.results[0]
+
+  if (results.error) {
+    return {errors: [results.error], series: []}
+  }
+
+  const series = results.series[0]
```

Comment:
is there a case where series is empty? if so, how would you feel about using a get to make sure that series[0] exists?

Reply:
👍